src/main/python/vantiqsdk.py

```diff
@@ -1437,10 +1452,12 @@ async def connect(self, do_pings: bool = True):
                                 self._vlog.debug('Message received via subscription.')
                                 callback = self.callbacks[request_id]
                                 await callback(self.MESSAGE, resp)
-            self.connected = False
 
-    async def subscribe(self, path: str, params: dict,
-                        callback: Callable[[str, dict], Awaitable[None]]) -> VantiqResponse:
+            # Once we get here, any transient subscriptions will be gone, so we should reset our side as well
+            await self.unsubscribe_all()
```

Comment:
This is what happens when the server disconnects entirely.  When it comes back the client may want to re-reinitialize without having to discard everything.  This makes sure that the state is clean so we can.